miso-web/src/spring-test/java/uk/ac/bbsrc/tgac/miso/webapp/springtest/QcStatusRestControllerST.java

```diff
@@ -0,0 +1,79 @@
+package uk.ac.bbsrc.tgac.miso.webapp.springtest;
+
+import org.junit.Test;
+import org.springframework.web.servlet.*;
+import static org.springframework.test.web.servlet.setup.MockMvcBuilders.*;
+
+import java.util.Arrays;
+import java.util.List;
+
+import javax.ws.rs.core.MediaType;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.*;
+import org.springframework.test.web.servlet.ResultActions;
+import uk.ac.bbsrc.tgac.miso.dto.QcNodeDto;
+import uk.ac.bbsrc.tgac.miso.core.data.impl.view.qc.SampleQcNode;
+
+
+import uk.ac.bbsrc.tgac.miso.dto.Dtos;
+import uk.ac.bbsrc.tgac.miso.dto.QcDto;
+import static org.hamcrest.Matchers.*;
+
+import static org.junit.Assert.*;
+import org.springframework.test.web.servlet.MockMvc;
+
+
+public class QcStatusRestControllerST extends AbstractST {
+
+  private static final String CONTROLLER_BASE = "/rest/qcstatuses";
+  private static final Class<SampleQcNode> entityClass = SampleQcNode.class;
+  // there's library, pool, and sample QC nodes (and others)
+  // they are largely the same, so for endpoint testing,
+  // it's acceptable to just test SampleQCs
```

Comment:
These affect completely different entities. They should definitely all be tested.